### ISSUE TITLE
Adicione ao F-Droid

### ISSUE THREAD

**Issue by pizzamaker** [RioBus/android#30]
O F-Droid é um catálogo de aplicativos livres para Android: https://f-droid.org/

Aqui estão as diretrizes: https://f-droid.org/contribute/

**Comment by pizzamaker** [comment]
Tópico no F-Droid: https://f-droid.org/forums/topic/riobus/

**Comment by pizzamaker** [comment]
@marco-jardim Algum sucesso?

**Comment by fmsouza** [comment]
@pizzamaker o cadastro desse site parece estar com algum problema, pois ele te pede pra clicar no link de "recuperar a senha", no e-mail, para poder cadastrá-la ao criar a conta. No entanto, sempre que tentamos atualizar a senha, aparece uma mensagem dizendo que o token é inválido.

**Comment by pizzamaker** [comment]
Você pode tentar pelo OpenID. De qualquer forma, aqui estão as linhas gerais de como incluir o app no F-Droid: https://f-droid.org/contribute/

**Comment by fmsouza** [comment]
@pizzamaker Estou tentando isso agora. Obrigado! Vou dar uma olhada.

**Comment by pizzamaker** [comment]
Tô doido pra usar esse aplicativo (o Arthur me falou sobre ele), mas não tenho Play Services no meu celular. O aplicativo depende do Play Services? Seria bacana oferecer um .apk avulso enquanto o app não é disponibilizado pelo F-Droid.

**Comment by fmsouza** [comment]
@pizzamaker Sim, é necessário ter a Play Services instalada sim por conta do mapa da Google. Boa dica! Quando rolar a próxima release aqui no Github podemos começar a disponibilizar o APK por aqui. Só não conseguimos disponibilizar o app avulso agora porque a versão que estamos trabalhando vem com um conjunto de mudanças muito grande, e tem vários conflitos com a versão atual do servidor. Quando atualizarmos, vamos atualizar tudo ao mesmo tempo. Vai ser um momento bastante importante!

**Comment by pizzamaker** [comment]
Não seria possível usar os mapas do OpenStreetMap ao invés do Google Maps? https://www.openstreetmap.org/

**Comment by fmsouza** [comment]
Estamos analisando a viabilidade do uso do OSM já tem algum tempo, só que pretendemos implementar algumas funcionalidades no futuro que precisam de informações sobre o trânsito, e apenas o Google Maps tem essa informação. Caso descubramos que essa informação não é tão relevante para escrever o algoritmo, iremos dar prioridade à troca do Google Maps pelo OSM.

**Comment by pizzamaker** [comment]
O problema é que, por mais que o app em si seja livre, o Play Services é uma dependência não livre, o que infelizmente torna o projeto em si não livre, e o inviabiliza pra muita gente que depende do F-Droid.

Gostaria de sugerir pelo menos uma versão livre do app, com OSM. E uma versão com Google Maps, se quiserem. Vai ajudar bastantão :)

**Comment by marco-jardim** [comment]
O uso da geolocalização sempre vai exigir o play services independente de usar o OSM ou não. Uma alternativa possível seria ter um release paralelo usando algo tipo o NOGAPPS (http://forum.xda-developers.com/showthread.php?t=1715375), ou então remover totalmente a geolocalização do app.

**Comment by pizzamaker** [comment]
Por mim, se não usar dependências privativas já tá mais do que ótimo :)
